src/tokenizer.rs

```diff
@@ -447,12 +447,21 @@ impl<'a> State<'a> {
 pub struct Tokenizer<'a> {
     dialect: &'a dyn Dialect,
     query: &'a str,
+    options: &'a TokenizerOptions,
+}
+
+pub struct TokenizerOptions {
+    pub no_escape: bool,
```

Comment:
perhaps we can add some comments here explaining what this is doing

Reply:
Thanks! I added the comment. 😊
29cb1e2